README.CentOS7.md

```diff
@@ -0,0 +1,154 @@
+# LORIS CentOS 7.x Notes
+
+This document contains supplemental details on how to perform a basic CentOS 7.x install of LORIS.
+Note that the main README in LORIS assumes that LORIS is being run on Ubuntu.
+
+This Readme assumes you already understand basic UNIX, MariaDB (MySQL) and Apache setup and
+settings. If you're not already comfortable troubleshooting system administration issues,
+you should not follow this guide.
+
+For further details on the install process, please see the LORIS GitHub Wiki CentOS Install page.  
+
+# System Requirements - Install dependencies
+
+Default dependencies installed by CentOS 7.x may not meet the version requirements for LORIS deployment or development:
+* MariaDB 10.2 is supported for LORIS 21.* 
```

Comment:
```suggestion
* MariaDB 10.3 is supported for LORIS 21.* 
```
as indicated by @driusan

Reply:
thanks @PapillonMcGill we'll fix this typo

Reply:
@christinerogers Ok!